src/sql/src/statements/describe.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// SQL structure for `DESCRIBE TABLE`.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct DescribeTable {
+    pub table_name: String,
```

Comment:
Also include catalog name and schema name in this struct.

Reply:
> Thanks for the feedback. I'll fix this soon

Currently I'm Ok with the catalog/schema name issue and marked these are resolved. Please fix other comments and we can merge this PR as soon as possible. Don't hesitate to reach us if you need any help!